neuralpredictors/data/datasets/movies.py

```diff
@@ -239,42 +239,56 @@ def transformed_mean(self, stats_source=None):
 class NRandomSubSequenceDataset(Dataset):
     """
     Data augmentation for training.
-    Generate a new dataset based on dat2, by random sampling of each training item in dat2 for multiple times.
-    This only works for movie data and each sampling is a subsequence of the full sequence in a dat2 item.
+    Generate a new dataset based on original_dat, by random sampling of each training item in original_dat for multiple times.
+    This only works for movie data and each sampling is a subsequence of the full sequence in a original_dat item.
     Args:
-        dat2: an original dataset
-        newtiers: list, tiers for each item in new dataset
-        newinds: list, indice for each item in new dataset
-        random_start: array, start positions at each dat2 item for random sampling
+        original_dat: an original dataset
+        new_tiers: list, tiers for each item in new dataset
```

Comment:
could we add this 2 things optional, so that if one do not pass it - it would be generated somehow by default?

Reply:
done